web/src/grammars/lua-mta.tmLanguage.json

```diff
@@ -10,6 +10,10 @@
       "match": "\\b(setCursorPosition|setCursorAlpha|getCursorPosition|getCursorAlpha)\\b",
       "name": "support.function.mta-client"
     },
+    {
```

Comment:
Since this file is generated on build, it could be ignored from GitHub. What do you think?